src/Agent/NewRelic/Agent/Core/Config/Configuration.xsd

```diff
@@ -2235,6 +2235,70 @@
                         </xs:all>
                     </xs:complexType>
                 </xs:element>
+
+                <xs:element name="opentelemetry" minOccurs="0" maxOccurs="1">
+                    <xs:annotation>
+                        <xs:documentation>
+                            OpenTelemetry configuration for the hybrid agent. This section contains configuration options
+                            that control OpenTelemetry signal processing and export to New Relic.
+                        </xs:documentation>
+                    </xs:annotation>
+
+                    <xs:complexType>
+                        <xs:all>
+                            <xs:element name="metrics" minOccurs="0" maxOccurs="1">
+                                <xs:annotation>
+                                    <xs:documentation>
+                                        Configuration for OpenTelemetry metrics collection and export.
+                                    </xs:documentation>
+                                </xs:annotation>
+
+                                <xs:complexType>
+                                    <xs:attribute name="enabled" type="xs:boolean" default="false">
+                                        <xs:annotation>
+                                            <xs:documentation>
+                                                Set this to "true" to enable OpenTelemetry metrics collection and export to New Relic.
+                                                This feature that must be used together with the global opentelemetry enabled setting.
+                                                Environment variable: NEW_RELIC_OPENTELEMETRY_METRICS_ENABLED
+                                            </xs:documentation>
+                                        </xs:annotation>
+                                    </xs:attribute>
+
+                                    <xs:attribute name="include" type="xs:string" use="optional">
+                                        <xs:annotation>
+                                            <xs:documentation>
+                                                A comma-delimited list of OpenTelemetry Meter names to include for metrics export to New Relic.
+                                                If specified, only meters matching these names will have their metrics exported.
```

Comment:
Should we clarify that this setting is used to override the built-in exclusion list? It does limit the meters to just the ones defined on. the inclusion list.